src/components/toast/toast.component.js

```diff
@@ -122,6 +133,8 @@ Toast.propTypes = {
   open: PropTypes.bool,
   /** Callback for when dismissed. */
   onDismiss: PropTypes.func,
+  /** Time for Toast to remain on screen */
+  timeout: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
   /** props used with flash component. Allow to center a component */
```

Comment:
The description references the flash component, I think that bit should be removed and instead it should be something like (Allows the toast to be centred) 
Also on line 145 there is a ( missing before the word see.